src/main_menu.cpp

```diff
@@ -13,8 +13,8 @@ bool mm_visible;
 
 Line lines[LINES] = {
     {"cheats", CHEAT_INDEX, "Turn cheats on/off", false},
-    {"flags", FLAGS_INDEX, "Turn in-game flags on/off", false},
-    {"inventory", INVENTORY_INDEX, "Set Link's items and equipment", false},
+    {"flags", FLAGS_INDEX, "Turn in game flags on/off", false},
```

Comment:
Still don't think there should be any changes to this file. Link was capitalized because it is a name, and in-game should have a dash because it is a compound adjective in this case. You can use "git checkout <branch> -- <path to this file>" to get the version of this file from a different branch.